desktop/libs/indexer/src/indexer/api3.py

```diff
@@ -731,9 +732,20 @@ def upload_local_file(request):
   upload_file = request.FILES['file']
   username = request.user.username
   filename = "%s_%s:%s;" % (username, uuid.uuid4(), upload_file.name)
+  file_format = upload_file.name.split(".")
```

Comment:
Nit: Since the name itself is `file_format`, then better to do `upload_file.name.split(".")[-1]` here only?

Reply:
done